python/nano/src/bigdl/nano/deps/inference/pytorch/model.py

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2016 The BigDL Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from bigdl.nano.pytorch.lightning import LightningModuleFromTorch
+import torch
+
+from .model_utils import get_forward_args
+
+
+class AcceleratedLightningModule(LightningModuleFromTorch):
```

Comment:
Let's move this out of deps, since this code does not introduce new dependencies.